### PR TITLE
LOG4J2-1997 This cleans up the entire test package of import wildcards…

### PR DESCRIPTION
**Pull request by kpm1985**
…, deprecated methods, typos and other trivial issues.

### CODE REVIEW

File: log4j-core/src/test/java/org/apache/logging/log4j/core/AppenderRefLevelJsonTest.java [hunk 1]
```diff
@@ -16,19 +16,20 @@
  */
 package org.apache.logging.log4j.core;
 
+import static org.hamcrest.Matchers.hasSize;
+import static org.junit.Assert.assertThat;
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/AppenderRefLevelJsonTest.java]: Static imports go last. This fix is incorrect.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/AppenderRefLevelTest.java [hunk 2]
```diff
@@ -16,6 +16,8 @@
  */
 package org.apache.logging.log4j.core;
 
+import static org.junit.Assert.assertEquals;
+
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/AppenderRefLevelTest.java]: Static imports go last. This fix is incorrect. We also allow wildcard static imports so this change isn't required.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/BasicLoggingTest.java [hunk 3]
```diff
@@ -28,7 +29,7 @@
     @Test
     public void test1() {
         final Logger logger = LogManager.getLogger(BasicLoggingTest.class.getName());
-        logger.debug("debug not set");
+        logger.debug("Debug not set");
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/BasicLoggingTest.java]: Why is the message being changed?

File: log4j-core/src/test/java/org/apache/logging/log4j/core/CronRolloverApp.java [hunk 4]
```diff
@@ -20,7 +20,7 @@
 import org.apache.logging.log4j.Logger;
 
 /**
- * Created by rgoers on 1/1/17.
+ * Created by Ralph Goers on 1/1/17.
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/CronRolloverApp.java]: This line should be removed entirely.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/CustomLevelsOverrideTest.java [hunk 5]
```diff
@@ -83,9 +83,9 @@ public void testLog() {
         final Logger logger = context.getLogger();
         final List<LogEvent> events = listAppender.getEvents();
         assertThat(events, hasSize(0));
-        logger.debug("Hello, {}", "World");
+        logger.debug("Hello, {}", "World Debug");
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/CustomLevelsOverrideTest.java]: Why is the message being changed?

File: log4j-core/src/test/java/org/apache/logging/log4j/core/CustomLevelsOverrideTest.java [hunk 6]
```diff
@@ -83,9 +83,9 @@ public void testLog() {
...
         assertThat(events, hasSize(1));
-        logger.log(warnLevel, "Hello DIAG");
+        logger.log(warnLevel, "Hello World Warn");
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/CustomLevelsOverrideTest.java]: Why is the message being changed?

File: log4j-core/src/test/java/org/apache/logging/log4j/core/DeadlockTest.java [hunk 7]
```diff
@@ -17,6 +17,7 @@
 package org.apache.logging.log4j.core;
 
 import org.apache.logging.log4j.junit.LoggerContextRule;
+
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/DeadlockTest.java]: We don't require whitespace here.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/CustomLevelsWithFiltersTest.java [hunk 8]
```diff
@@ -25,6 +25,7 @@
 import org.apache.logging.log4j.core.filter.CompositeFilter;
 import org.apache.logging.log4j.core.filter.ThresholdFilter;
 import org.apache.logging.log4j.junit.LoggerContextRule;
+
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/CustomLevelsWithFiltersTest.java]: We don't require whitespace here. We don't disallow it either.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/CustomLevelsTest.java [hunk 9]
```diff
@@ -26,6 +26,7 @@
 import org.apache.logging.log4j.Level;
 import org.apache.logging.log4j.junit.LoggerContextRule;
 import org.apache.logging.log4j.test.appender.ListAppender;
+
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/CustomLevelsTest.java]: We don't require whitespace here. We don't disallow it either, but not sure what the point is.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/ExtendedLevelTest.java [hunk 10]
```diff
@@ -22,6 +22,7 @@
 import org.apache.logging.log4j.junit.LoggerContextRule;
 import org.apache.logging.log4j.test.ExtendedLevels;
 import org.apache.logging.log4j.test.appender.ListAppender;
+
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/ExtendedLevelTest.java]: We don't require whitespace here. We don't disallow it either, but not sure what the point is.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/GcFreeAsynchronousLoggingTest.java [hunk 11]
```diff
@@ -18,6 +18,7 @@
 
 import org.apache.logging.log4j.categories.AsyncLoggers;
 import org.apache.logging.log4j.core.async.AsyncLoggerContextSelector;
+
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/GcFreeAsynchronousLoggingTest.java]: We don't require whitespace here. We don't disallow it either, but not sure what the point is.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/GcFreeSynchronousLoggingTest.java [hunk 12]
```diff
@@ -32,6 +32,7 @@ public void testNoAllocationDuringSteadyStateLogging() throws Throwable {
 
     /**
      * This code runs in a separate process, instrumented with the Google Allocation Instrumenter.
+     * First is the above spelled correctly, secondly where is the import then? (KM)
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/GcFreeSynchronousLoggingTest.java]: Why is this in the patch? It should be a question on the dev list.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/HostNameTest.java [hunk 13]
```diff
@@ -16,21 +16,24 @@
  */
 package org.apache.logging.log4j.core;
 
+import static org.hamcrest.Matchers.endsWith;
+import static org.hamcrest.Matchers.hasSize;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/HostNameTest.java]: static imports go last. We also allow wildcard imports of static imports so this change isn't required.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/CollectionLoggingTest.java [hunk 14]
```diff
@@ -26,6 +26,7 @@
 import org.apache.logging.log4j.junit.LoggerContextRule;
 import org.apache.logging.log4j.message.StringMapMessage;
 import org.apache.logging.log4j.test.appender.ListAppender;
+
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/CollectionLoggingTest.java]: We don't require whitespace here.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/HostNameTest.java [hunk 15]
```diff
@@ -16,21 +16,24 @@
...
 import java.util.List;
 
 import org.apache.logging.log4j.core.appender.RollingFileAppender;
 import org.apache.logging.log4j.core.util.NetUtils;
 import org.apache.logging.log4j.junit.LoggerContextRule;
 import org.apache.logging.log4j.test.appender.ListAppender;
 import org.apache.logging.log4j.util.Strings;
+
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/HostNameTest.java]: We don't require whitespace here. We don't disallow it either, but not sure what the point is.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/LateConfigTest.java [hunk 16]
```diff
@@ -26,6 +26,7 @@
 import org.apache.logging.log4j.core.config.DefaultConfiguration;
 import org.apache.logging.log4j.core.config.xml.XmlConfiguration;
 import org.apache.logging.log4j.status.StatusLogger;
+
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/LateConfigTest.java]: We don't require whitespace here. We don't disallow it either, but not sure what the point is.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/LevelTest.java [hunk 17]
```diff
@@ -16,6 +16,12 @@
  */
 package org.apache.logging.log4j.core;
 
+import static org.hamcrest.Matchers.hasSize;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/LevelTest.java]: We allow wildcards on static imports so this change isn't necessary.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/LogEventFactoryTest.java [hunk 18]
```diff
@@ -34,6 +34,7 @@
 import org.apache.logging.log4j.junit.LoggerContextRule;
 import org.apache.logging.log4j.message.Message;
 import org.apache.logging.log4j.test.appender.ListAppender;
+
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/LogEventFactoryTest.java]: We don't require whitespace here. We don't disallow it either, but not sure what the point is.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/LogEventFactoryTest.java [hunk 19]
```diff
@@ -52,7 +53,7 @@
 
     private ListAppender app;
 
-    // this would look so cool using lambdas
+    // consider using lambdas
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/LogEventFactoryTest.java]: Why is this change required?

File: log4j-core/src/test/java/org/apache/logging/log4j/core/LogEventFactoryTest.java [hunk 20]
```diff
@@ -87,7 +88,7 @@ public void before() {
     @Test
     public void testEvent() {
         final org.apache.logging.log4j.Logger logger = context.getLogger("org.apache.test.LogEventFactory");
-        logger.error("error message");
+        logger.error("Error message");
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/LogEventFactoryTest.java]: Why does this message need to be changed?

File: log4j-core/src/test/java/org/apache/logging/log4j/core/LogEventTest.java [hunk 21]
```diff
@@ -27,6 +27,7 @@
 import org.apache.logging.log4j.core.impl.Log4jLogEvent;
 import org.apache.logging.log4j.message.Message;
 import org.apache.logging.log4j.message.SimpleMessage;
+
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/LogEventTest.java]: We don't require whitespace here. We don't disallow it either, but not sure what the point is.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/LogEventTest.java [hunk 22]
```diff
@@ -99,20 +100,20 @@ public void testNanoTimeIsNotSerialized1() throws Exception {
         final LogEvent actual = (LogEvent) ois.readObject();
         assertNotEquals("Different event: nanoTime", copy, actual);
         assertNotEquals("Different nanoTime", copy.getNanoTime(), actual.getNanoTime());
-        assertEquals("deserialized nanoTime is zero", 0, actual.getNanoTime());
+        assertEquals("Deserialized nanoTime is zero", 0, actual.getNanoTime());
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/LogEventTest.java]: Why does this message need to be changed?

File: log4j-core/src/test/java/org/apache/logging/log4j/core/LoggerDateTest.java [hunk 23]
```diff
@@ -16,6 +16,8 @@
  */
 package org.apache.logging.log4j.core;
 
+import static org.junit.Assert.assertTrue;
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/LoggerDateTest.java]: static imports go last. We allow wildcard imports so this change isn't necessary.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/LoggerUpdateTest.java [hunk 24]
```diff
@@ -16,11 +16,15 @@
  */
 package org.apache.logging.log4j.core;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertSame;
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/LoggerUpdateTest.java]: static imports go last. We allow wildcard static imports so this change isn't necessary.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/LookupTest.java [hunk 25]
```diff
@@ -16,14 +16,15 @@
  */
 package org.apache.logging.log4j.core;
 
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/LookupTest.java]: static imports go last. We allow wildcard static imports so this change isn't necessary.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/PatternSelectorTest.java [hunk 26]
```diff
@@ -16,6 +16,10 @@
  */
 package org.apache.logging.log4j.core;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/PatternSelectorTest.java]: static imports go last. We allow wildcard static imports so this change isn't necessary.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/PropertiesFileConfigTest.java [hunk 27]
```diff
@@ -24,6 +24,7 @@
 
 import org.apache.logging.log4j.core.config.Configuration;
 import org.apache.logging.log4j.junit.LoggerContextRule;
+
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/PropertiesFileConfigTest.java]: We don't require whitespace here.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/PropertiesFileConfigTest.java [hunk 28]
```diff
@@ -52,7 +53,7 @@ public void testReconfiguration() throws Exception {
         final File file = new File(CONFIG);
         final long orig = file.lastModified();
         final long newTime = orig + 10000;
-        assertTrue("setLastModified should have succeeded.", file.setLastModified(newTime));
+        assertTrue("setLastModified() should have succeeded.", file.setLastModified(newTime));
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/PropertiesFileConfigTest.java]: why do we need the parens?

File: log4j-core/src/test/java/org/apache/logging/log4j/core/ShutdownDisabledTest.java [hunk 29]
```diff
@@ -16,13 +16,14 @@
  */
 package org.apache.logging.log4j.core;
 
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/ShutdownDisabledTest.java]: static imports go last. We allow wildcard static imports so this change isn't necessary.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/ShutdownTimeoutConfigurationTest.java [hunk 30]
```diff
@@ -16,13 +16,14 @@
  */
 package org.apache.logging.log4j.core;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/ShutdownTimeoutConfigurationTest.java]: static imports go last. We allow wildcard static imports so this change isn't necessary.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/StrictXmlConfigTest.java [hunk 31]
```diff
@@ -16,6 +16,8 @@
  */
 package org.apache.logging.log4j.core;
 
+import static org.junit.Assert.assertEquals;
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/StrictXmlConfigTest.java]: static imports go last. We allow wildcard static imports so this change isn't necessary.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/TimestampMessageTest.java [hunk 32]
```diff
@@ -16,6 +16,9 @@
  */
 package org.apache.logging.log4j.core;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/TimestampMessageTest.java]: static imports go last. We allow wildcard static imports so this change isn't necessary.

File: log4j-core/src/test/java/org/apache/logging/log4j/core/XmlEvents.java [hunk 33]
```diff
@@ -23,6 +23,7 @@
 import org.apache.logging.log4j.junit.LoggerContextRule;
 import org.apache.logging.log4j.message.AsynchronouslyFormattable;
 import org.apache.logging.log4j.message.StructuredDataMessage;
+
```
Comment (rgoers) [inline log4j-core/src/test/java/org/apache/logging/log4j/core/XmlEvents.java]: We don't require whitespace here.

**Comment by kpm1985** [comment]
And thanks for your input, but I think my time would be better served somewhere else where people don't refer the COD for calling out some childish commenting and capital letters. Take care!
